modules/cert/main.tf

```diff
@@ -0,0 +1,13 @@
+resource "google_compute_managed_ssl_certificate" "frontendcert" {
+  provider = google-beta
+
+  name = "${var.prefix}-cert-${var.install_id}"
```

Comment:
I think this can be changed to:

name = "${var.prefix}cert-${var.install_id}"

To avoid adding two '-' characters

Reply:
oh yes, I thought there might be some of those hanging out. I'll fix this, thanks.